analysis/neutron/openmc_model.py

```diff
@@ -133,6 +133,8 @@ def baby_geometry(x_c: float, y_c: float, z_c: float):
         + cover
         + z_c
     )
+    z_plane_14 = openmc.ZPlane(z_c - 28.00)
+    z_plane_15 = openmc.ZPlane(z_c - 28.00 - epoxy_thickness)
```

Comment:
I see this 28.00 magic number in a lot of places... maybe it's worth having a variable for it?